src/plugins/cordova-plugin-inappbrowser/app-host-handlers.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+module.exports = function () {
+    var Browser = require('./inapp-browser'),
+        instance;
+
+    function execute(fnName) {
+        if (!instance) {
+            return;
+        }
+
+        var fn = instance[fnName],
+            args;
+
+        if (Object.keys(arguments).length > 1) {
+            // remove the first element since it is the "fnName" parameter
+            Array.prototype.shift.apply(arguments);
+            args = arguments;
+        }
+
+        return fn.apply(instance, args);
+    }
+
+    return {
+        'InAppBrowser': {
```

Comment:
The latest docs show a slightly different set of methods [here](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-inappbrowser/#inappbrowser):

 
-    addEventListener
-    removeEventListener
-    close
-    show
-    executeScript
-    insertCSS